review/static/README.txt

```diff
@@ -0,0 +1,19 @@
+This static directory is for files that should be included in your kit as plain
+static files.
```

Comment:
I'm guessing this was an auto-generated file. I would probably delete it for cleanliness.